indico/web/templates/forms/dropzone_widget.html

```diff
@@ -15,7 +15,7 @@
             </div>
         </div>
     </div>
-    <input class="change-trigger" type="hidden" value="0" name="__file_change_trigger">
+    <input class="change-trigger" type="hidden" value="{{ 1 if field.data else 0}}" name="__file_change_trigger">
```

Comment:
I think the only important thing is to know whether the user removed the file (set 'change-trigger' to 0), added a new one (set it to a different value from the current one, e.g. 2) or kept the current one (1).
If there is no file, then we only need two states: either there's a new one (2) or not (0). If there's not a new one, we want the button to be disabled anyway.